sdk/go/x/auto/test/errors/runtime_error/dotnet/MyStack.cs

```diff
@@ -0,0 +1,13 @@
+using Pulumi;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // out of bounds exception
+        #pragma warning disable
+        string[] arr = null;
+        #pragma warning disable
+        var x = arr[0];
```

Comment:
You could do something simple like `throw new ApplicationException("thrown for testing");`